linter/lints/cabf_br/lint_crl_validity_period.go

```diff
@@ -42,17 +47,73 @@ func (l *crlValidityPeriod) CheckApplies(c *x509.RevocationList) bool {
 }
 
 func (l *crlValidityPeriod) Execute(c *x509.RevocationList) *lint.LintResult {
-	validity := c.NextUpdate.Sub(c.ThisUpdate)
-	if validity <= 0 {
+	/*
+		Let's Encrypt issues CRLs for two distinct purposes:
+		   1) CRLs containing subscriber certificates created by the
+		      crl-updater. These CRLs must have only the distributionPoint and
+		      onlyContainsUserCerts fields set.
+		   2) CRLs containing subordinate CA certificates created by the
+		      ceremony tool. These CRLs must only have the onlyContainsCACerts
+		      field set.
+
+		RFC 5280 Section 5.2.5
+
+		IssuingDistributionPoint ::= SEQUENCE {
+			distributionPoint          [0] DistributionPointName OPTIONAL,
+			onlyContainsUserCerts      [1] BOOLEAN DEFAULT FALSE,
+			onlyContainsCACerts        [2] BOOLEAN DEFAULT FALSE,
+			...
+		}
```

Comment:
nit: this block comment feels slightly out of place for _this_ lint (it's perfect for the other lint). Here, I'd maybe strip it down to something like:
> Let's Encrypt issues two kinds of CRLs:
>1) CRLs containing subscriber certificates, created by crl-updater. These assert the onlyContainsUserCerts boolean.
>2) CRLs containing issuer CRLs, created by the ceremony tool. These assert the onlyContainsCACerts boolean.
>
> We use the presence of these booleans to determine which BR-mandated lifetime to enforce.